### PR TITLE
change from const to static-assets for the html and css assets dir

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

File: lib/src/html_generator.dart [hunk 1]
```diff
@@ -5,9 +5,8 @@
 library dartdoc.html_generator;
 
 import 'dart:io';
-
+import 'dart:developer';
```
Comment (devoncarew) [inline lib/src/html_generator.dart]: Is there a dependency on a version of the sdk for this? If so, we should update our sdk dep.

Comment (sethladd) [inline lib/src/html_generator.dart]: good catch! I just saw the warning that dart:profiler was deprecated.

I'll update the SDK dep.

**Comment by sethladd** [comment]
@devoncarew bumped SDK constraint

**Comment by devoncarew** [comment]
lgtm!

**Comment by devoncarew** [comment]
Hmm, we'll need to take out stable bot offline until 1.11is stable.

File: pubspec.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 description: A documentation generator for Dart.
 homepage: https://github.com/dart-lang/dartdoc
 environment:
-  sdk: '>=1.9.0 <2.0.0'
+  sdk: '>=1.11.0 <2.0.0'
```
Comment (devoncarew) [inline pubspec.yaml]: also, it looks like the lower bounds will need to be a dev build - something like `1.11.0-dev.5.2`

**Comment by sethladd** [comment]
Reverted back to 1.9 for now. It's just a hint from analyzer, so for now, no big deal. Once 1.11 lands, we can switch over to dart:developer

**Comment by sethladd** [comment]
Fixed with b9860e3

**Pull request closed without merging**